### ISSUE TITLE
Calling SyncedCron.start() in multiple packages

### ISSUE THREAD

**Issue by SachaG** [percolatestudio/meteor-synced-cron#34]
When using synced-cron in multiple packages, how should you call `SyncedCron.start()`?

It seems like calling it one time in each package makes each job run as many times as there are packages, which is obviously not what we want. 

Is the only option to call it once from the core app itself?

**Comment by zol** [comment]
Yes, this is an issue. I really should be checking `SyncedCron.running` in the `start` function and preventing the re-scheduling of jobs.

**Comment by SachaG** [comment]
Similarly, it seems like setting `SyncedCron.options` in one package affects the others as well? Or am I wrong about that?

**Comment by zol** [comment]
No, that's correct. `SyncedCron` is designed as a singleton so options are global.

**Comment by SachaG** [comment]
Can't that be a problem if different packages are setting different options?

**Comment by zol** [comment]
Yep :) We'd accept a PR that changes `SyncedCron` to a class rather than a singleton. It's only a singleton for historical reasons, namely we didn't think too much about potential uses when building the package.

**Comment by em0ney** [comment]
@SachaG I deal with this in a project by having a base package that initialises SyncedCron, exports it from its Package scope.  Then all packages that use SyncedCron define that base package as a dependency.

**Comment by Ethaan** [comment]
Is there a way to call an specific Sync Task just one time?, im facing the same issue.

I have an application with a package structure, and i have 3 differents Synced Tasks, so i have 3 

      SyncedCron.start();

The problem over here is that 1 Synced Task is for sending email, and the tasks is sending the same email 3 times, is there a way to fix this?

**Comment by em0ney** [comment]
Hi Ethan,

There is no need to initialise SyncedCron 3 times.  After the first time it is initialised, if you add a new scheduled job using `SyncedCron.add`, the job will be added to the schedule.

In terms of executing a scheduled job just once - I'd recommend simply de-scheduling the task at the end of your scheduled job's code.

Cheers,
Elliott
